tgstation.dme

```diff
@@ -3195,7 +3195,6 @@
 #include "code\modules\reagents\chemistry\holder.dm"
 #include "code\modules\reagents\chemistry\reagents.dm"
 #include "code\modules\reagents\chemistry\recipes.dm"
-#include "code\modules\reagents\chemistry\machinery\alchemy_table.dm"
```

Comment:
don't just uninclude this file, delete it so people don't get confused in the future

Reply:
Done deleted, and gone.